lib/stats-collector.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+/**
+ * Stats Collector
+ *
+ * Provides stats such as test duration,
+ * number of tests passed / failed etc.
+ *
+ * @public
+ * @api public
```

Comment:
Delete `@api` line - just use standard JSDoc - this was referenced by one of TJ's old packages.